index.json

```diff
@@ -9,5 +9,11 @@
     "description": "Map of the Virtual States of the ERTH2 Universe",
     "geoURL": "https://erth2.github.io/movc/geo/geo.geojson",
     "countryInfoURL": "https://erth2.github.io/movc/geo/src/countries.json"
+  },
+  "vgovernments": {
+    "name": "Virtual Governments",
+    "description": "Map of the Virtual States of the ERTH2 Universe",
```

Comment:
The information does not correspond to reality

Reply:
I changed the description.